standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -18,7 +18,7 @@
 package org.apache.hadoop.hive.metastore.txn;
 
 import com.google.common.annotations.VisibleForTesting;
-import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.collections4.CollectionUtils;
```

Comment:
Just thinking out loud, I don't see any commons-collections-3.x dependency present in standalone-metastore module, so commons-collections-3.x must be coming from some transitive dependency (either from hadoop/atlas/commons-beanutils), now in this PR we will start shipping commons-collections-4.x in `standalone-metastore/metastore-server/target/apache-hive-metastore-4.1.0-SNAPSHOT-bin`. Can it cause any  classpath issue?